daemonize.py

```diff
@@ -161,17 +161,22 @@ def start(self):
             # We will continue with syslog initialization only if actually have such capabilities
             # on the machine we are running this.
             if os.path.exists(syslog_address):
-                syslog = handlers.SysLogHandler(syslog_address)
-                if self.verbose:
-                    syslog.setLevel(logging.DEBUG)
-                else:
-                    syslog.setLevel(logging.INFO)
-                # Try to mimic to normal syslog messages.
-                formatter = logging.Formatter("%(asctime)s %(name)s: %(message)s",
-                                              "%b %e %H:%M:%S")
-                syslog.setFormatter(formatter)
-
-                self.logger.addHandler(syslog)
+                try:
+                    syslog = handlers.SysLogHandler(syslog_address)
+                    if self.verbose:
+                        syslog.setLevel(logging.DEBUG)
+                    else:
+                        syslog.setLevel(logging.INFO)
+                    # Try to mimic to normal syslog messages.
+                    formatter = logging.Formatter("%(asctime)s %(name)s: %(message)s",
+                                                  "%b %e %H:%M:%S")
+                    syslog.setFormatter(formatter)
+
+                    self.logger.addHandler(syslog)
+                except Exception, e:
+                   print("syslog service may not work, please check and start syslog-ng or specified custom logger in Daemonize", str(e)) 
```

Comment:
`syslog-ng` is just one of many possible syslog daemons out there. Please make the error message more generic.